setup.cfg

```diff
@@ -1,6 +1,21 @@
+[bdist_wheel]
+universal=1
+
 [metadata]
+license_file = COPYING.LESSER
```

Comment:
What is this?  (the file does not exist)

Reply:
copy paste ;-)

Reply:
Please fix it.